origins_workflow/origins_workflow.routing.yml

```diff
@@ -1,4 +1,14 @@
 
+origins_workflow.moderation_state_controller_new_draft_of_published:
+  path: '/origins_workflow/new_draft_of_published/{nid}'
+  defaults:
+    _form: '\Drupal\origins_workflow\Form\NewDraftOfPublishedForm'
+    _title: 'New draft of published'
+  requirements:
+    _permission: 'access content'
```

Comment:
This permission would allow anyone the ability to create a new draft just be adding a number to the URL

Reply:
Great spot, thanks !